src/reactviews/common/utils.ts

```diff
@@ -20,3 +26,14 @@ export function formatString(str: string, ...args: any[]): string {
     }
     return result;
 }
+
+export const getVscodeTheme = (theme: Theme) => {
```

Comment:
For functions that aren't just local functions, it's more readable to use the normal `export function getVscodeThemeType() { }` syntax over `export const getVsCodeThemeType = ...`